Error when trying to generate blueprints

hi, i got error when trying this blueprint

```
$ ember g authenticator oauth2 --base-class=oauth2 --pod
version: 2.3.0
Cannot find module 'silent-error'
Error: Cannot find module 'silent-error'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/run/media/rizkytahara/c65441ef-b53a-4a79-8d05-3349fd1e332c/web/dev/biscuit/bas/node_modules/ember-simple-auth/blueprints/authenticator/index.js:2:19)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
```

any idea how to solve that? Thank you...

--- a/package.json
+++ b/package.json
@@ -58,7 +58,6 @@
     "mocha-only-detector": "0.0.2",
     "morgan": "^1.5.2",
     "rimraf": "^2.3.2",
-    "silent-error": "^1.0.0",
     "sinon-chai": "~2.8.0",
     "torii": "~0.6.1"
   },
@@ -67,7 +66,8 @@
   ],
   "dependencies": {
     "ember-cli-babel": "^5.1.5",
-    "ember-getowner-polyfill": "1.0.0"
+    "ember-getowner-polyfill": "1.0.0",
+    "silent-error": "^1.0.0"
   },
   "ember-addon": {
     "configPath": "tests/dummy/config"

